src/main/java/club/issizler/okyanus/api/cmdnew/ArgumentType.java

```diff
@@ -0,0 +1,20 @@
+package club.issizler.okyanus.api.cmdnew;
+
+public enum ArgumentType {
```

Comment:
If you're keeping this class, shouldn't you just... not deprecate the one in the old package?